### DISCUSSION TITLE
Package distribution

### DISCUSSION THREAD

**Discussion by kadykov** [donniebreve/touchcursor-linux#68] [Ideas]
First of all, thank you all for this amazing application, it changes my way to interact with the keyboard.
Do you plan to distribute it as a package, for instance, a flatpack? In this case, it will be accessible to a wider audience.

**Comment by donniebreve** [comment]
I don't currently have any plans to distribute this project as a flatpak package. From my (limited) understanding, packaging services or daemons using flatpak/appimage/snap is not easy. This application requires some elevated permissions to capture the keyboard and I'm not sure that's possible with those distribution methods.

Someone is already packaging this project for the AUR and, despite my attempt to reach out, they're continuing things on their own. That may be an option for you, and I hope people find the AUR package useful.

If anyone would like to try to package this in flatpak, please do!

**Reply by kadykov** [reply]
Unfortunately, it is not an option for me because I want to use touchcursor on Fedora Silverblue.
What do you think about making an `rpm` package?

**Reply by donniebreve** [reply]
I think it's a decent idea. Would you like to take a shot at creating a workflow that creates the package? I guess this would force me to figure out versioning and releases.
